model_zoo/tests/data_reader_tests/jag_single_layer_ae.prototext

```diff
@@ -4,10 +4,7 @@ model {
   serialize_io: true
```

Comment:
This prototext does not have a trainer block. Now it would be empty, but other files have empty blocks.